workshop.wwwapi/Models/Appointment.cs

```diff
@@ -3,12 +3,20 @@
 namespace workshop.wwwapi.Models
 {
     //TODO: decorate class/columns accordingly
+    [Table("appointments")]
     public class Appointment
     {
+        [Column("id")]
+        public int Id { get; set; }
```

Comment:
no need of this since you're specifying a different COMPOSITE primary key in your modelBuilder